astropy/io/ascii/rst.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under a 3-clause BSD style license
+"""
+:Author: Simon Gibbons (simongibbons@gmail.com)
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from .core import DefaultSplitter
+from .fixedwidth import (FixedWidth,
+                         FixedWidthData,
+                         FixedWidthHeader,
+                         FixedWidthTwoLineDataSplitter)
+
+class SimpleRSTHeaderSplitter(DefaultSplitter):
+    delimiter = ' '
+
+class SimpleRSTHeader(FixedWidthHeader):
+    position_line = 0
+    start_line = 1
+    splitter_class = SimpleRSTHeaderSplitter
+    position_char = '='
+
+class SimpleRSTDataSplitter(FixedWidthTwoLineDataSplitter):
```

Comment:
I think you can avoid making this custom splitter class by the following (not tested):
```
class SimpleRSTHeader(FixedWidthHeader):
    # (stuff you already have)
    def get_cols(self, lines):
        super(SimpleRSTHeader, self).get_cols(lines)

        # Right hand column can be unbounded
        self.cols[-1].end = None
```